src/index.js

```diff
@@ -0,0 +1,178 @@
+let temperatureLabel = null;
+let temperatureIncreaseButton = null;
+let temperatureDecreaseButton = null;
+let getRealtimeTemperatureButton = null;
+let skyImage = null;
+let landscapeImage = null;
+let cityNameInput = null;
+let cityNameResetButton = null;
+let skySelectValue = null;
+let gardenBackground = null;
+let temperatureValue = null;
+let defaultCityName = null;
+
+const showTemp = () => {
```

Comment:
`updateTemperature` updates the `textContent` as well as propogating the relevant changes to the color and landscape. Prefer using `updateTemperature` consistently and removing `showTemp`.